library/include/rocblas-functions.h

```diff
@@ -1736,31 +1742,43 @@ rocblas_qgemm_strided_batched(
+                                                            rocblas_int                  ldb,
+                                                            rocblas_int                  stride_b,
+                                                            const rocblas_float_complex* beta,
+                                                            rocblas_float_complex*       C,
+                                                            rocblas_int                  ldc,
+                                                            rocblas_int                  stride_c,
+                                                            rocblas_int batch_count);
 
-ROCBLAS_EXPORT rocblas_status
-rocblas_zgemm_strided_batched(
-    rocblas_handle handle,
-    rocblas_operation transa, rocblas_operation transb,
-    rocblas_int m, rocblas_int n, rocblas_int k,
-    const rocblas_double_complex *alpha,
-    const rocblas_double_complex *A, rocblas_int lda, rocblas_int stride_a,
-    const rocblas_double_complex *B, rocblas_int ldb, rocblas_int stride_b,
-    const rocblas_double_complex *beta,
-          rocblas_double_complex *C, rocblas_int ldc, rocblas_int stride_c,
-    rocblas_int batch_count );
-*/
+ROCBLAS_EXPORT rocblas_status rocblas_zgemm_strided_batched(rocblas_handle                handle,
+                                                            rocblas_operation             transa,
+                                                            rocblas_operation             transb,
+                                                            rocblas_int                   m,
+                                                            rocblas_int                   n,
+                                                            rocblas_int                   k,
+                                                            const rocblas_double_complex* alpha,
+                                                            const rocblas_double_complex* A,
+                                                            rocblas_int                   lda,
+                                                            rocblas_int                   stride_a,
+                                                            const rocblas_double_complex* B,
+                                                            rocblas_int                   ldb,
+                                                            rocblas_int                   stride_b,
+                                                            const rocblas_double_complex* beta,
+                                                            rocblas_double_complex*       C,
+                                                            rocblas_int                   ldc,
+                                                            rocblas_int                   stride_c,
+                                                            rocblas_int batch_count);
```

Comment:
For gemm_ex documentation at line 1876 need to add information on rocblas_datatype_f32_c and rocblas_datatype_f64r to "Supported types are as follows:"

Reply:
modified in commit "add document for gemm_ex and correct func format"